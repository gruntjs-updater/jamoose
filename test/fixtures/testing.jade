html
  head
    style.
      h1 {
        color: red;
        font-weight: bold;
      }

  body
    h1 Testing

    p lorem ipsum