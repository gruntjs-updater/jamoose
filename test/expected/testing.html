<html><head></head><body><h1 style="color: red; font-weight: bold;">Testing</h1><p>lorem ipsum</p></body></html>
